src/styles/blocks/header.scss

```diff
@@ -0,0 +1,158 @@
+@import "../utils/mixins";
+@import "../utils/vars";
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  grid-template-rows: 1fr 1fr;
+  align-items: start;
+
+  scroll-behavior: smooth;
```

Comment:
'scroll-behavior' should be applied to the body or html selector, not in .header.

Reply:
I deleted this atribute.